docs/components/MessagingAttachmentView.jsx

```diff
@@ -0,0 +1,264 @@
+import * as React from "react";
+
+import Example, { CodeSample, ExampleCode } from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import { MessagingAttachment, FlexBox } from "src";
+import { FileAttachmentIcon } from "src/MessagingAttachment/MessagingAttachment";
+
+import "./MessagingAttachmentView.less";
+
+const cssClass = {
+  CONFIG_CONTAINER: "MessagingAttachmentView--configContainer",
+  CONFIG_OPTIONS: "MessagingAttachmentView--configOptions",
+  CONFIG: "MessagingAttachmentView--config",
+  CONFIG_LABEL_TEXT: "TextInput2View--configLabelText",
+  CONFIG_TOGGLE: "MessagingAttachmentView--configToggle",
+  CONTAINER: "MessagingAttachmentView",
+  INTRO: "MessagingAttachmentView--intro",
+  PROPS: "MessagingAttachmentView--props",
+  DEMO_ROW: "MessagingAttachmentView-demoRow",
+};
+
+// TODO: FINISH THIS DOCUMENT
```

Comment:
What's left to be finished, out of curiosity?

Reply:
nada